scss/components/dropdown.scss

```diff
@@ -130,7 +130,7 @@ $block: #{$fd-namespace}-dropdown;
             @if $fd-support-css-var-fallback {
               box-shadow: 0 0 0 1px fd-color-state("hover", "action");
             }
-            box-shadow: 0 0 0 1px var(--fd-color-action-hover);
+            box-shadow: none;
```

Comment:
This is getting rid of the box shadow on the button itself when its popover is displayed, right?

Reply:
@greg-a-smith Yes, exactly.